caikit_nlp/modules/text_generation/peft_prompt_tuning.py

```diff
@@ -300,6 +300,8 @@ def train(
         torch_dtype: Optional[str] = None,  # TODO: Optional[Union[torch.dtype, str]]
         silence_progress_bars: Optional[bool] = True,
         seed: int = RANDOM_SEED,
+        train_on_completion: bool = False,
+        response_template: str = None,
```

Comment:
decision to expose this flag may change, just fYI. I wont be merging this code right away, but I want to ensure rest of it looks ok, assuming it is a user provided argument